elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -1384,6 +1384,15 @@ elytron.client-ssl-context.ssl-session.peer-certificates.version=The certificate
 # Operations
 elytron.client-ssl-context.ssl-session.invalidate=Invalidate the SSLSession (Note: This does not terminate current connections, only prevents future connections from joining or resuming this session).
 
+elytron.dynamic-client-ssl-context=An Dynamic SSLContext for use on the client side of a connection. It allows to choose ssl context based on peer's host and port information
+# operations
+elytron.dynamic-client-ssl-context.add=Add the Dynamic Client SSLContext definition.
+elytron.dynamic-client-ssl-context.remove=Remove the Dynamic Client SSLContext definition.
+#Attributes
+elytron.dynamic-client-ssl-context.authentication-context=The authentication context that will be queried for ssl context based on peer information.
+elytron.dynamic-client-ssl-context.active-session-count=The count of current active sessions.
```

Comment:
Since `hasActiveSessions` always returns `false`, is `active-session-count` always going to be 0? If so, it might be worth pointing that out here since something similar has been pointed out for `ssl-session` below.

Reply:
@fjuma active-session-count is determined by calling of the methods sslContext.getServerSessionContext() and sslContext.getClientSessionContext()  if serverOrHostController is true. 

But since these methods are throwing an exception for dynamic SSL context , I changed serverOrHostController to be always false and removed active session count from LocalDescriptions.properties.